vars/pipelineVegavisorManageNode.groovy

```diff
@@ -114,6 +115,10 @@ void call() {
                             command: "validator join --node ${SHORT_NODE} --get-eth-to-submit-bundle",
                             returnStdout: true,
                         ).trim()
+                        ANSIBLE_VARS_DICT = [
+                            'healthcheck_type': 'time_check',
+                            'healthcheck_timeout': 3600 // 1h to follow the network
```

Comment:
One hour is too long. On all of our networks, most of the time, it takes up to 2min. The only longer ones are experiencing issues that should be reported (they never move a block, so increasing timeout even to 1 day won't help), the other situation is when the grpc servers are unhealthy, which should not happen to us.

What you have probably experienced with your test is that the network was just started, and there was no checkpoint yet. I don't think the network behaves correctly in this situation. 

Either way, setting it to something like 3-5min should be enough.